### ISSUE TITLE
Add no-magic-numbers rule

### ISSUE THREAD

**Issue by sergeysova** [atomixinteractions/eslint-config-base#2]
https://github.com/airbnb/javascript/blob/5aa203eaa88fb7febfb1ccf8736e896ca0959049/packages/eslint-config-airbnb-base/rules/best-practices.js#L137

**Comment by sergeysova** [comment]
```diff
-var value = input.size * 0.283
+const COERS_COEF = 0.283
+var value = input.size * COERS_COEF
```

**Comment by sergeysova** [comment]
Resolved in 04dcf16
